itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/BaseReplicationScenariosAcidTables.java

```diff
@@ -98,6 +98,7 @@ static void internalBeforeClassSetup(Map<String, String> overrides, Class clazz)
       put("hive.metastore.disallow.incompatible.col.type.changes", "false");
       put("hive.in.repl.test", "true");
       put("metastore.warehouse.tenant.colocation", "true");
+      put(HiveConf.ConfVars.REPL_DATA_COPY_LAZY.varname, "false");
```

Comment:
We can set it in WarehouseInstance itself, it will act as a default for both the test repl base classes.
Additionally in Test files which doesn't extends from BaseReplicationScenario*.java like you have in TestReplicationScenarios.java if any.

Reply:
The other tests don't need this flag. Adding it here works fine. Tests are green.